v1/atlas_test.go

```diff
@@ -118,6 +121,49 @@ func (hs *atlasServer) authenticationHandler(w http.ResponseWriter, r *http.Requ
 	}
 }
 
+func (hs *atlasServer) tfConfigLatest(w http.ResponseWriter, r *http.Request) {
+	if r.Method != "GET" {
+		w.WriteHeader(http.StatusMethodNotAllowed)
+		return
+	}
+
+	fmt.Fprintf(w, `
+	{
+		"version": {
+			"version": 5,
+			"metadata": { "foo": "bar" }
```

Comment:
We should probably return `variables` here too, just so we can test that the client reads them properly